packages/playwright-test/src/reporters/merge.ts

```diff
@@ -57,18 +72,21 @@ function parseEvents(reportJsonl: Buffer): JsonEvent[] {
   return reportJsonl.toString().split('\n').filter(line => line.length).map(line => JSON.parse(line)) as JsonEvent[];
 }
 
-async function extractAndParseReports(dir: string, shardFiles: string[]): Promise<{ metadata: BlobReportMetadata, parsedEvents: JsonEvent[] }[]> {
+async function extractAndParseReports(dir: string, shardFiles: string[], printStatus: StatusCallback): Promise<{ metadata: BlobReportMetadata, reportFile: string, parsedEvents: JsonEvent[] }[]> {
   const shardEvents = [];
   await fs.promises.mkdir(path.join(dir, 'resources'), { recursive: true });
   for (const file of shardFiles) {
+    printStatus(`extracting: ${file}`);
     const zipFile = new ZipFile(path.join(dir, file));
     const entryNames = await zipFile.entries();
     for (const entryName of entryNames) {
+      printStatus(`extracting: ${file} > ${entryName}`);
```

Comment:
I would drop this, since it's a lot of spam.

Reply:
I did have it in the beginning and it looked weird when I didn't see what was being instructed